DateRangeBox: add the Overview demo description (#2615)
DevExpress/devextreme-demos: JSDemos/Demos/DateRangeBox/Overview/description.md

> The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to enter or pick dates in a specified range. This demo illustrates the following DateRangeBox properties:

Reviewer: ```suggestion
The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to select a date range. The component includes input boxes for start/end dates and a date-picker dropdown. This demo shows how you can use the following DateRangeBox settings:
```

> The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to select a date range. The component includes input boxes for start/end dates and a date-picker dropdown. This demo shows how you can use the following DateRangeBox settings:

Reviewer 2: ```suggestion
The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to select a date range. The component includes input boxes for start/end dates and a drop-down date picker. This demo shows how you can use the following DateRangeBox settings:
```

- [value](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#value)     

> An array where you can specify the selected range (start and end dates). The DateRangeBox allows you to define the start and end dates separately. For this purpose, use the [startDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#startDate) and [endDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#endDate) properties.

Reviewer: ```suggestion
An array where you can specify the selected range (start and end dates). The DateRangeBox also allows you to define the start and end dates separately. For this purpose, use the [startDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#startDate) and [endDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#endDate) properties instead.
```

- [displayFormat](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#displayFormat)        
A date display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter.

- [disabled](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#disabled)        
Specifies whether the DateRangeBox responds to user interaction.

- [showClearButton](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#showClearButton)        

> This property allows you to display the button that clears the DateRangeBox **values**.

Reviewer: Other descriptions don't start with "this property", so let's continue the same pattern for consistency. 

 ```suggestion
Displays the button that clears DateRangeBox **values**.
```

- [multiView](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#multiView)     

> Allows allows you to pick the start or end dates in a single or multiple calendars.

Reviewer: ```suggestion
Switches between a single-month and two-month dropdown calendar.
```
